### DISCUSSION TITLE
Show DateRangePicker without clicking the input

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [rsuite/rsuite#3760] [Q&A]
Hi, 
I would like to display the `DateRangePicker` by default on page load.
I'd like to keep the input hidden and only update the value on change. 

Is that possible in the current version?

Thanks

**Comment by simonguo** [comment]
If you want the DateRangePicker to open the picker by default, you can set an `open` prop. 
```tsx
<DateRangePicker open />
```
Do you want something like `Calendar`? Currently not supported.
